internal/pkg/cli/pipeline_show.go

```diff
@@ -185,7 +185,7 @@ func (o *showPipelineOpts) askPipelineName() error {
 
 func (o *showPipelineOpts) retrieveAllPipelines() ([]string, error) {
 	pipelines, err := o.pipelineSvc.ListPipelineNamesByTags(map[string]string{
-		"ecs-project": o.ProjectName(),
+		"copilot-application": o.ProjectName(),
```

Comment:
I wonder if we should pull the names of these tags somewhere else? A const somewhere. I can create an issue for it and we can tackle that later.

Reply:
good catch!